File: Leap_Year_Checker.c
#include<stdio.h>
LeapChecker(int year);
int main(){
    int year,c;
    printf("Enter the year which is to be checked: ");
    scanf("%d",& year);
    c=LeapChecker(year);
    if(c==1)
    printf("%d is a Leap Year", year);
    if(c==0)
    else
    printf("%d is not a Leap Year", year);
    return 0;
}
    LeapChecker(int year)
    {
    int leap;
    leap=year%4;
    if(leap==0)
    leap=1;
    //else
    //leap=0;
    return leap;
    }

File: LeapYearChecker.java
import java.util.Scanner;
public class LeapYearChecker{
    public static void main(String[] args){
        Scanner inp=new Scanner(System.in);
        int year,c;
        System.out.println("Enter the Year which is to be checked as a Leap Year: ");
        year=inp.nextInt();
        if(year%4==0)
        System.out.println(year+" is a Leap Year");
        else
        System.out.println(year+" is not a Leap Year") 
    }
}    
